### ISSUE TITLE
Could not able to load the data.

### ISSUE THREAD

**Issue by JafferWilson** [crate/crate-commoncrawl#6]
I am trying to work with Common Crawl file and came across Crate.io.
I am following the instructions as specified in the instruction part.
I am having trouble with this command:
`COPY commoncrawl FROM 'ccrawl://cr8.is/1WSiodP';`
Getting the following error:
```
 COPY ERROR (0.000 sec)
Error!

SQLActionException[MalformedURLException: unknown protocol: ccrawl] 
```
I have also traced the issue. Here is the trace:
```
SQLActionException: INTERNAL_SERVER_ERROR 5000 MalformedURLException: unknown protocol: ccrawl
	at java.net.URL.<init>(URL.java:600)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at java.net.URI.toURL(URI.java:1089)
	at io.crate.operation.collect.files.URLFileInput.getStream(URLFileInput.java:52)
	at io.crate.operation.collect.files.FileReadingCollector.readLines(FileReadingCollector.java:228)
	at io.crate.operation.collect.files.FileReadingCollector.doCollect(FileReadingCollector.java:205)
	at io.crate.operation.collect.MapSideDataCollectOperation$1$1.run(MapSideDataCollectOperation.java:135)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```
Kindly help me.
Here is the screen shot of the issue:
![screenshot from 2016-12-01 15-07-16](https://cloud.githubusercontent.com/assets/13446197/20788767/160c7662-b7d8-11e6-84ca-0c4c7bbeb336.png)

**Comment by celaus** [comment]
Hi, there :) 

Can you see if you have any server logs of CrateDB? 

There should be some lines like that in there:

``` 
[2016-11-26 17:17:17,298][INFO ][node                     ] [Mont Aiguille] version[2.4.1], pid[5661], build[${build/NA]
[2016-11-26 17:17:17,298][INFO ][node                     ] [Mont Aiguille] initializing ...
[2016-11-26 17:17:17,521][INFO ][io.crate.plugin          ] [Mont Aiguille] plugins loaded: [crate-sigar, crate-commoncrawl]
... 
``` 

Cheers, Claus

**Comment by JafferWilson** [comment]
Please check the following log files. I have checked it but do not understand the problem.

[crate.log.2016-12-03.txt](https://github.com/crate/crate-commoncrawl/files/636025/crate.log.2016-12-03.txt)
[crate.log.2016-12-02.txt](https://github.com/crate/crate-commoncrawl/files/636027/crate.log.2016-12-02.txt)
[crate.log.2016-12-01.txt](https://github.com/crate/crate-commoncrawl/files/636026/crate.log.2016-12-01.txt)

**Comment by celaus** [comment]
Hi @JafferWilson 

Looking at the logs there are 2 things I can see: 
- you need to distribute the plugin to ALL nodes in order to import on all nodes
- You seem to have placed the jar in multiple locations, which leads to the JarHell exception you are getting in log nr 2. remove this: jar1: /usr/share/crate/lib/crate-commoncrawl-1.3-SNAPSHOT.jar

**Comment by JafferWilson** [comment]
Kindly, let me know how I can distribute plugin to all nodes. I think I only have one node right now. Still let me know.

**Comment by celaus** [comment]
What's your mode of deployment? The easiest way is to use rsync or ftp/http transfer and retrieve the plugin from a centralized source.

**Comment by JafferWilson** [comment]
I am new to crate, claus. I do not know what exactly you mean by mode of deployment. Currently the crate is running on my local system. If you can show me how I can distribute the plugin to the node.. it will help a lot. And I am using only one node.

**Comment by celaus** [comment]
Mode of deployment would mean docker and the likes :) 

on a local install, just copy the jar into the plugins folder in your installation directory. The JarHell exception is due to the plugin being available more than once :)

**Comment by JafferWilson** [comment]
Thank you, claus. I think this helps..... :)

**Comment by JafferWilson** [comment]
Just a small query: I got this error:

```
Error!

OutOfMemoryError[Java heap space] 
```
Can you tell me how I can get through this.
I have just executed this command in the console:
`COPY commoncrawl FROM 'ccrawl://cr8.is/1WSiodP';`

**Comment by celaus** [comment]
have you set the CRATE_HEAP_SPACE variable correctly? CrateDB's copy from is a very "unlocked" mechanism of importing data, so it won't do any major checking :) (default is 1g I think) 


https://crate.io/docs/reference/configuration.html#environment-variables

**Comment by JafferWilson** [comment]
@celaus  I tried your configuration link. after configuring Crate.yml, Crate is not getting started. When I reset everything crate was working fine.

**Comment by celaus** [comment]
Hi, @JafferWilson the environment variables should be set in the environment, not in the crate.yml .

**Comment by JafferWilson** [comment]
@celaus Glad to know from you regarding the configuration. Actually, I had figured it out already after I read the crate.yml@celaus Glad to know from you regarding the configuration. Actually, I had figured it out already after I read the crate.yml file. And I have set the respective variables in my system.
I really appreciate your help. I am glad that I can now work and perform my experiments.
Thank you once again. I am closing this issue, fully satisfied. :)
